src/pages/Admin/Charity/Permissions/MobileTable.tsx

```diff
@@ -0,0 +1,143 @@
+import { FormField, FormValues, UpdateableFormValues } from "./schema";
+
+const formValues: UpdateableFormValues = {
+  accountFees: {} as FormField,
+  beneficiaries_allowlist: {} as FormField,
+  contributors_allowlist: {} as FormField,
+  donationSplitParams: {} as FormField,
+  profile: {} as FormField,
+};
+const FORM_KEYS = getTypedKeys(formValues);
+
+export default function MobileTable({ className = "", disabled = false }) {
+  const { endow_type } = useAdminResources<"charity">();
+  const { wallet } = useGetWallet();
+  const {
+    watch,
+    formState: { errors },
+  } = useFormContext<FormValues>();
+
+  return (
+    <div className={`${className} border border-prim rounded-t`}>
+      <div className="grid items-center grid-cols-[auto_1fr] h-12 uppercase text-xs font-bold bg-orange-l6 dark:bg-blue-d7 border-b border-prim divide-x divide-prim rounded-t">
+        <div className="w-12" />
+        <div className="py-3 px-4">Action</div>
+      </div>
+
+      {FORM_KEYS.map((fieldName) => (
+        <Disclosure
+          key={`mob-table-row-${fieldName}`}
+          as="div"
+          className="text-sm odd:bg-orange-l6 dark:even:bg-blue-d6 dark:odd:bg-blue-d7 w-full border-b last:border-0 border-prim"
+        >
+          {({ open }) => {
+            const delegate = watch(`${fieldName}.delegate`);
+            const name = watch(`${fieldName}.name`);
+            const initDelegate = watch(`initialValues.${fieldName}.delegate`);
+            const initDelegateAddress = watch(
+              `initialValues.${fieldName}.delegate_address`
+            );
+            const initModifiable = watch(
+              `initialValues.${fieldName}.modifiable`
+            );
+            const modifiable = watch(`${fieldName}.modifiable`);
+
+            const isDisabled =
+              disabled ||
+              !initModifiable ||
+              (initDelegate &&
+                !!initDelegateAddress &&
+                initDelegateAddress !== wallet?.address);
+
+            const inputDisabled = isDisabled || !modifiable;
+
+            return (
+              <>
+                <Disclosure.Button
+                  className={`${
+                    open ? "bg-orange-l5 dark:bg-blue-d4" : ""
+                  } w-full grid grid-cols-[auto_1fr_auto] divide-x divide-prim`}
+                >
+                  <DrawerIcon
+                    size={24}
+                    className={`${
+                      open ? "text-orange" : ""
+                    } w-12 place-self-center`}
+                    isOpen={open}
+                  />
+                  <p className="text-sm p-4 text-left h-full truncate">
+                    {name}
+                  </p>
+                </Disclosure.Button>
+                <Disclosure.Panel className="w-full font-work divide-y divide-prim border-t border-prim">
+                  <div className="grid gap-4 py-3 px-4">
+                    <CheckField<FormValues>
```

Comment:
There's a bug with checkboxes when switching between desktop and mobile, it will be addressed in https://github.com/orgs/AngelProtocolFinance/projects/2/views/1?pane=issue&itemId=23101212